docs/nightly/en/user-guide/operations/disaster-recovery/dr-solution-based-on-cross-region-deployment-in-single-cluster.md

```diff
@@ -1 +1,87 @@
-# DR Solution Based on Cross-Region Deployment in a Single Cluster
\ No newline at end of file
+# DR Solution Based on Cross-Region Deployment in a Single Cluster
+
+## How Disaster Recovery Works in GreptimeDB
+GreptimeDB is well-suited for cross-region disaster recovery. Customers may have varying regional characteristics and business needs, and GreptimeDB offers tailored solutions to meet these diverse requirements.
+
+GreptimeDB resource management involves the concept of AZs (Availability Zones). An AZ is a logical unit of disaster recovery.
+It can be a DC(Datra Center), a compartment of a DC. This depends on the customer's specific DC conditions and deployment design.
+
+A GreptimeDB region is a city. When two DC are in the same region and one DC becomes unavailable, the other DC can take over the services of the unavailable DC. This is a localization strategy.
```

Comment:
maybe we need to explain this region is different from the region in the datanode. Or use an another word instead of region here.

Reply:
In this context, the term "region" is the easiest for everyone to understand. To avoid confusion, I did not mention the concept of GretptimeDB table region in this document.